docs/developers-guide/e2e-tests.md

```diff
@@ -127,3 +127,31 @@ These are the tags currently in use:
 
 - `@external` - tests that require an external docker container to run
 - `@actions` - tests that use metabase actions and mutate data in a data source
+
+## How to stress-test a flake fix?
+
+Fixing a flaky test locally doesn't mean the fix works in GitHub's CI environment. The only way to be sure the fix works is to stress-test it in CI. That's what `.github/workflows/e2e-stress-test-flake-fix.yml` is made for. It allows you to quickly test the fix in your branch without waiting for the full build to complete.
+
+Please follow these steps:
+### Prepare
+- Create a new branch with your proposed fix and push it to the remote
+- Either skip opening a PR altogether or open a **draft** pull request
+
+### Obtain the artifact ID
+- Go to the latest successful commit on `master` branch
+- Click on the green checkmark next to that commit
+- Choose either "E2E Tests / build (ee)" job or "Build + Docker Uberjar / Build MB ee" job and click on the _Details_ link next to it (it will take you to that job's summary page within a related workflow)
+- Click on the workflow _Summary_
+- Scroll to the bottom of the page where you'll find the _Artifacts_ section that contains `metabase-oss-uberjar` and `metabase-ee-uberjar` artifacts
+- Right click on any of the two (but prefer EE one, unless you specifically need to test OSS changes) and copy its link
+- The link will look like this: `https://github.com/metabase/metabase/suites/13087680507/artifacts/710350560`
+- `710350560` is the artifact id that you'll need in the next step
+
+### Trigger the stress-test workflow manually
+- Go to `https://github.com/metabase/metabase/actions/workflows/e2e-flake-fix-stress-test.yml`
```

Comment:
@nemanjaglumac This URL doesn't work. The actual URL is https://github.com/metabase/metabase/actions/workflows/e2e-stress-test-flake-fix.yml